run.sh

```diff
@@ -12,14 +12,14 @@ if [ $startConnIdx -eq $# ]; then
   stressArgs=${*:2}
 else 
   connectorArgs=${*:$((startConnIdx))} 
-  sparkmaster=${*:$((startConnIdx-1)):1}
+  sparkmaster="--master ${*:$((startConnIdx-1)):1}"
```

Comment:
Right now we will provide the `--master` option only if the user provides it, otherwise we leave it out completely. Another approach we can take is to set the master default to `localhost` (not implemented here). Either approach is fine for me.